code/game/objects/items/spear.dm

```diff
@@ -149,7 +149,7 @@
 	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities among Nanotrasen military forces."
 	attack_verb_continuous = list("gores")
 	attack_verb_simple = list("gore")
-	force=15
+	force = 15
```

Comment:
why are we editing the space in between the = sign on the greytide??

Reply:
like it's just a weird entirely random change to make

Reply:
it looks nicer and is more consistent with the formatting of other code.

Reply:
`FastHub didn't allow me to message in time and I was too busy playing Serious Sam so didn't bother messaging here but.
Consistency in formatting yeah.`